playwright.config.ts

```diff
@@ -0,0 +1,77 @@
+import { defineConfig, devices } from '@playwright/test';
+import * as dotenv from 'dotenv';
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+dotenv.config();
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig({
+  testDir: './tests',
+  /* Run tests in files in parallel */
+  fullyParallel: true,
+  /* Fail the build on CI if you accidentally left test.only in the source code. */
+  //forbidOnly: !!process.env.CI,
+  /* Retry on CI only */
+  //retries: process.env.CI ? 2 : 0,
+  /* Opt out of parallel tests on CI. */
+  //workers: process.env.CI ? 1 : undefined,
+  /* Reporter to use. See https://playwright.dev/docs/test-reporters */
+  reporter: 'html',
+  /* Shared settings for all the projects below. See https://playwright.dev/docs/api/class-testoptions. */
+  use: {
+    /* Base URL to use in actions like `await page.goto('/')`. */
+    baseURL: process.env.HOME_LOCATION || 'http://127.0.0.1:4321',
```

Comment:
```suggestion
    baseURL: process.env.HOME_LOCATION || 'http://localhost:4321',
```

If we are defaulting we should default to localhost